xtext-website/community.html

```diff
@@ -373,14 +373,6 @@ <h2>What others have built with Xtext</h2>
 						<td>Cedric Vidal</td>
 					</tr>
 
-					<tr>
-						<td><a href="https://www.eclipse.org/incquery/">EMF&nbsp;IncQuery</a></td>
```

Comment:
do we have an entry for viatra?

Reply:
Yes:
![image](https://user-images.githubusercontent.com/20393472/83106656-221fe480-a0bd-11ea-86e9-3bcc4f67203b.png)